### ISSUE TITLE
As a CLI User I want to run the CLI on windows without the Windows Linux Subsystem

### ISSUE THREAD

**Issue by ryw** [astronomer/astro-cli#167]

**Comment by andriisoldatenko** [comment]
@vparekh94 could you please share minimum windows version we are going to support?

**Comment by schnie** [comment]
I'm thinking a first attempt at this would only need to support Windows 10, since the goal is only to support users avoiding the Linux Subsystem, which is only available on 10. Not sure if any customers are waiting on this or not. @vparekh94 would know.

**Comment by andriisoldatenko** [comment]
I build exe file with:
`GOOS=windows GOARCH=386 CGO_ENABLED=0 go build -o astro-win main.go`

Known issues:
- `astro airflow init` didn't create correct folder structure.

**Comment by andriisoldatenko** [comment]
tl;dr: we can close ticket, it works on Windows, more details in slack #windows channel.
```
C:\Users\IEUser\demo>astro-cli deployment list
 NAME       DEPLOYMENT NAME              ASTRO      DEPLOYMENT ID
 andrii     ecliptical-totality-5975     v0.7.5     cjt4h94o70f970f521eri2v7m

C:\Users\IEUser\demo>astro-cli airflow deploy
Authenticated to alpha.datarouter.ai

Select which airflow deployment you want to deploy to:
 #     LABEL      DEPLOYMENT NAME              WORKSPACE                         DEPLOYMENT ID
 1     andrii     ecliptical-totality-5975     Andrii Soldatenko's Workspace     cjt4h94o70f970f521eri2v7m

> 1
Deploying: ecliptical-totality-5975
ecliptical-totality-5975/airflow
Building image...
Sending build context to Docker daemon  12.29kB
Step 1/1 : FROM astronomerinc/ap-airflow:master-1.10.2-onbuild
# Executing 5 build triggers
 ---> Using cache
 ---> Using cache
 ---> Using cache
 ---> Using cache
 ---> Using cache
 ---> 05afc011311f
Successfully built 05afc011311f
Successfully tagged ecliptical-totality-5975/airflow:latest
SECURITY WARNING: You are building a Docker image from Windows against a non-Windows Docker host. All files and directories added to build context will have '-rwxr-xr-x' permissions. It is recommended to double check and reset permissions for sensitive files and directories.
Pushing image to Astronomer registry
The push refers to repository [registry.alpha.datarouter.ai/ecliptical-totality-5975/airflow]
a3120a5713e5: Preparing
3389e7f7b76b: Preparing
b5bb1f02d9b9: Preparing
b79e6d083090: Preparing
0290268f693d: Preparing
254d7ac9c2be: Waiting
1eb72bd80deb: Waiting
7f6f16343969: Waiting
1a5a6ad42684: Waiting
c2f4f17c8b58: Waiting
3ebe9911e207: Waiting
a464c54f93a9: Waiting
unauthorized: No authorization credentials specified
Error: command 'docker push registry.alpha.datarouter.ai/ecliptical-totality-5975/airflow:cli-1' failed: failed to execute cmd: exit status 1
2019/04/14 12:51:31 command 'docker push registry.alpha.datarouter.ai/ecliptical-totality-5975/airflow:cli-1' failed: failed to execute cmd: exit status 1
```

**Comment by paolaperaza** [comment]
@andriisoldatenko Nice. Can you paste step-by-step instructions on how a user might set this up in here (I was thinking Slack earlier, but might as well memorialize those instructions here).

From there, I can format everything and work on this [this Docs Issue](https://github.com/astronomer/docs/issues/59) for our Windows CLI install guide.

**Comment by andriisoldatenko** [comment]
@paolaperaza @schnie https://gist.github.com/andriisoldatenko/e351f5310d14c0270fad681bfd7c49d3
